crates/symbolicator/Cargo.toml

```diff
@@ -40,7 +40,8 @@ serde = { version = "1.0.119", features = ["derive", "rc"] }
 serde_json = "1.0.61"
 serde_yaml = "0.8.15"
 structopt = "0.3.21"
-symbolic = { git = "https://github.com/getsentry/symbolic", branch = "fix/demangle-fixes", version = "8.5.0", features = ["common-serde", "debuginfo", "demangle", "minidump-serde", "symcache"] }
+# NOTE: Keep symbolic in sync with symsorter
+symbolic = { git = "https://github.com/getsentry/symbolic", branch = "fix/demangle-fixes", version = "8.6.1", features = ["common-serde", "debuginfo", "demangle", "minidump-serde", "symcache"] }
```

Comment:
👍 this then also needs to be removed from symsorter.

I was wondering about this, actually. @Swatinem was this dependency missed after our conversation? Symbolic was updated multiple times in the last weeks, yet the version number remained here.

Reply:
That line was last touched ~3 month ago; otherwise it was just lockfile bumps. Sounds like cargo does not care too much about the `version` after all?

Reply:
As long as it's in the semver range, I think. If the branch head bumps to, say v9, it'll will start complaining in `cargo update` and the likes.

Reply:
Ah absolutely, forgot about that :-D